client/src/components/common/applicant-info-2.vue

```diff
@@ -260,8 +260,8 @@ export default class ApplicantInfo2 extends Vue {
         await newReqModule.postNameRequests('draft')
       } else {
         if (!this.editMode && ['COND-RESERVE', 'RESERVED'].includes(this.nrState)) {
-          let request = await newReqModule.getNameRequest(nrId)
-          if (request.stateCd === 'CANCELLED') {
+          const request = await newReqModule.getNameRequest(nrId)
+          if (request?.stateCd === 'CANCELLED') {
```

Comment:
Definitely no action required now, but another instance of where we can create an enum to clean up the strings 👍